test/common.js

```diff
@@ -64,18 +66,23 @@ function rmdirSync(p, originalEr) {
 }
 
 exports.refreshTmpDir = function() {
+  try {
+    fs.accessSync(testRoot, fs.W_OK);
+  } catch (e) {
+    if (e.code === 'ENOENT') {
+      fs.mkdirSync(testRoot);
```

Comment:
Yeah, that was my original approach (bail and let the user create it) by @joaocgreis suggested I'd try and create it. Should I invoke `mkdirp` instead of `mkdirSync`?